precompile/contracts/bibliophile/api.go

```diff
@@ -0,0 +1,135 @@
+package bibliophile
+
+import (
+	"math/big"
+
+	"github.com/ava-labs/subnet-evm/precompile/contract"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type VariablesReadFromClearingHouseSlots struct {
+	MaintenanceMargin  *big.Int         `json:"maintenance_margin"`
+	MinAllowableMargin *big.Int         `json:"min_allowable_margin"`
+	Amms               []common.Address `json:"amms"`
+}
+
+func GetClearingHouseVariables(stateDB contract.StateDB) VariablesReadFromClearingHouseSlots {
```

Comment:
Please also add `GetActiveMarketsCount`, `GetMarkets`, `GetNotionalPositionAndMargin`, `GetTotalNotionalPositionAndUnrealizedPnl`, `GetTotalFunding`, `GetUnderlyingPrices` to the api response. Even if you are not adding assertions on them in the test that's fine but I'd like to receive these fields so that we have the freedom to use them later.

Same for other contracts.